pkg/sloop/server/internal/config/config.go

```diff
@@ -172,16 +173,31 @@ func (c *SloopConfig) Validate() error {
 }
 
 func loadFromFile(filename string) *SloopConfig {
-	yamlFile, err := ioutil.ReadFile(filename)
-	if err != nil {
-		panic(fmt.Sprintf("failed to read %v. %v", filename, err))
-	}
-	var config SloopConfig
-	err = yaml.Unmarshal(yamlFile, &config)
-	if err != nil {
-		panic(fmt.Sprintf("failed to unmarshal %v. %v", filename, err))
+	if strings.Contains(filename,".yaml") {
+		yamlFile, err := ioutil.ReadFile(filename)
+		if err != nil {
+			panic(fmt.Sprintf("failed to read %v. %v", filename, err))
+		}
+		var config SloopConfig
```

Comment:
maybe better to declare this before if statement? so you don't need to define it twice (line 192).

Reply:
made the changes